src/Main.java

```diff
@@ -1,9 +1,43 @@
+import java.util.Scanner;
 
 public class Main {
     public static void main(String[] args) {
 
-        // Hier mag je je code schrijven voor de hoofd-opdracht
+        //integer array
+        Integer[] numeric = {1, 2, 3, 4, 5, 6, 7, 8, 9, 0};
+        String[] alphabetic = {"een", "twee", "drie", "vier", "vijf", "zes", "zeven", "acht", "negen", "tien", "nul"};
 
+        Translator translator = new Translator(alphabetic, numeric);
 
+        boolean play = true;
```

Comment:
Knap dat je doorhad dat die boolean nodig was voor ongeldige invoer. Dat was mij totaal onduidelijk wat ik met die boolean aanmoest. Ik heb het ook anders opgelost. De vraagstelling is soms zo onduidelijk .. of ligt dat aan mij?